app/controllers/audits_controller.rb

```diff
@@ -90,6 +90,13 @@ def destroy
 
   private
 
+  def handle_audit_errors
+    if @audit.errors.present?
+      errors = @audit.errors.collect { |error| "#{error.attribute.capitalize} ".tr("_", " ") + error.message }
+      flash[:error] = errors[1]
```

Comment:
This smells like a magic number - why are we getting only the second error?

Reply:
I grabbed the second one because that error message was the closest to what @cielf had said in the ticket that it should say. I am happy to display both of them if you would prefer.

Reply:
Relying on the order of errors is incredibly fragile. If you are trying to condense two similar errors into one, you might want to actually examine the strings for sentinel values and remove one if you find two of them.

Reply:
@dorner Heard! No problem 😉  Is this more like what you would like to see?
<img width="1159" alt="Screenshot 2023-07-06 at 3 08 19 PM" src="https://github.com/rubyforgood/human-essentials/assets/114014697/bc4c0ca8-794a-43e5-bdc3-4a070978512a">

Reply:
Hmm... not quite. What this is doing is picking out one specific error message (one that has "blank" in it) and only showing it if it exists, and swallowing all other messages.

I was thinking something like:

```ruby
flash[:error] = @audit.errors.uniq(&:attribute).map(&:message)
```

That *should* make it so that you only get one error message per attribute.

Reply:
Ahhhh! okay! I see! I was able to implement your suggestion while displaying the attribute of the error with the error message.

<img width="753" alt="Screenshot 2023-07-07 at 2 53 36 PM" src="https://github.com/rubyforgood/human-essentials/assets/114014697/5e8a2219-2e9d-4f25-97a3-f42e90a384fd">